cocos/core/geometry/sphere.ts

```diff
@@ -107,12 +105,12 @@ export class Sphere {
      * Set the components of a sphere to the given values
      * @zh
      * 将球体的属性设置为给定的值。
-     * @param {Sphere} out 接受操作的 sphere。
-     * @param cx 形状的相对于原点的 X 坐标。
-     * @param cy 形状的相对于原点的 Y 坐标。
-     * @param cz 形状的相对于原点的 Z 坐标。
-     * @param {number} r 半径。
-     * @return {Sphere} out 接受操作的 sphere。
+     * @param {Sphere} out @en Sphere which the properties set to. @zh 接受操作的 sphere。
```

Comment:
The sphere to set properties to
The sphere which the properties will be set to

Either is ok